utils/grpc/runtime_test.go

```diff
@@ -0,0 +1,56 @@
+package grpc
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/tron-us/go-btfs-common/config"
+	"github.com/tron-us/go-btfs-common/protos/shared"
+
+	"github.com/tron-us/go-common/v2/constant"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+)
+
+func init() {
+	if config.InitTestDB() != nil {
+	}
+}
+
+func TestRuntimeServer(t *testing.T) {
+
+	pgConMaps := map[string]string{"DB_URL_STATUS": config.DbStatusURL, "DB_URL_GUARD": config.DbGuardURL}
+	rdCon := config.RdURL
+
+	runtime := RuntimeServer{DB_URL: pgConMaps, RD_URL: rdCon, serviceName: "hub"}
+	report, err := runtime.CheckRuntime(context.Background(), &shared.SignedRuntimeInfoRequest{})
+	if err != nil {
```

Comment:
no need to `if err != nil` if you are asserting.

Reply:
okay.